rest/src/main/java/org/jboss/pnc/rest/endpoint/RunningBuildRecordEndpoint.java

```diff
@@ -49,14 +60,22 @@ public RunningBuildRecordEndpoint(BuildRecordProvider buildRecordProvider) {
         this.buildRecordProvider = buildRecordProvider;
     }
 
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successful retrieval all RunningBuildRecords"),
+            @ApiResponse(code = 204, message = "No RunningBuildRecords available"),
+    })
     @ApiOperation(value = "Gets all running Build Records")
     @GET
-    public List<BuildRecordRest> getAll(
+    public Response getAll(
```

Comment:
I'm not 100% but wouldn't returning a null do the job? This is mentioned in the spec as well as in StackOverflow: http://stackoverflow.com/questions/14102797/how-to-throw-http-204-status-code-using-jersey-framework-in-restful-web-service